### DISCUSSION TITLE
kpanda/user-guide/gpu/dynamic-regulation/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4496] [Announcements]
# kpanda/user-guide/gpu/dynamic-regulation/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/gpu/dynamic-regulation/

**Comment by samzong** [comment]
文档建议， 在修改了显存或者算力之后，如何观察出来变更

**Reply by lengrongfu** [reply]
已经添加通过 nvidia-smi 可看出资源变化。
